FiveCalls/FiveCalls/Extensions/Error+Offline.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Error+Offline.swift
+//  FiveCalls
+//
+//  Created by Ben Scheirman on 1/9/19.
+//  Copyright © 2019 5calls. All rights reserved.
+//
+
+import Foundation
+
+extension Error {
+    func isOfflineError() -> Bool {
+        let e = self as NSError
+        guard e.domain == NSURLErrorDomain else { return false }
+        
+        return e.code == NSURLErrorNetworkConnectionLost ||
```

Comment:
Just a style comment (you may or may not agree), but I find that 

    return [NSURLErrorNetworkConnectionLost, NSURLErrorNotConnectedToInternet, NSURLErrorSecureConnectionFailed].contains(e.code)

Is less code and more readable.

Reply:
Agreed, I find this style much more readable